paywall/src/__tests__/unlock.js/CheckoutUIHandler.test.ts

```diff
@@ -188,7 +188,7 @@ describe('CheckoutUIHandler', () => {
         PostMessages.UPDATE_TRANSACTIONS,
         fakeTransactions,
       ],
-      ['UPDATE_NETWORK', PostMessages.UPDATE_NETWORK, 3],
+      ['UPDATE_NETWORK', PostMessages.UPDATE_NETWORK, 1],
```

Comment:
This test update is just a consequence of using a tighter type on the network postmessage: `3` is not a valid network id in our apps.